lib/account/account-manager.js

```diff
@@ -1,102 +1,171 @@
 "use strict";
+
 /**
  * Manages the business logic of account controls.
  */
 
 /**
- * The object sent in which has information needed to validate the user
- * has properly set up their two factor authentication and has generated
- * a password for use to save.
+ * Contains information needed to validate the user has
+ * properly set up their two factor authentication and
+ * has generated a password.
+ *
+ * @typedef {Object} accountManager~accountinfo
```

Comment:
Typedef seems wrong to me.  Just the "accountinfo" bit (assuming you meant "accountInfo").  This is not an account, nor is it information about an account.  It is perhaps `accountManager~registrationInfo`.